packages/documentation/src/stories/foundation/layout/grid/grid.docs.mdx

```diff
@@ -20,7 +20,7 @@ import SampleScssMixins from './grid-scss-mixins.sample.scss?raw';
 
 Our grid system uses a series of containers, rows, and columns to layout and align content. It’s built with flexbox and is fully responsive. Below is an example and an in-depth explanation for how the grid system comes together.
 
-<Canvas of={GridStories.Basis} />
+<Canvas sourceState="shown" of={GridStories.Basis} />
```

Comment:
I would recommend expanding the source only on the first canvas on components, but leave foundation pages as they are.

Reply:
For the moment all the pages wil be have the source code shown. Maybe this wil change in the future